bsp4j/src/main/java/ch/epfl/scala/bsp4j/ScalaExtension.xtend

```diff
@@ -118,9 +118,11 @@ class ScalaMainClass {
   @NonNull @SerializedName("class") String className
   @NonNull List<String> arguments
   @NonNull List<String> jvmOptions
-  new(@NonNull String className, @NonNull List<String> arguments, @NonNull List<String> jvmOptions) {
+  @NonNull List<String> environmentVariables
```

Comment:
I don't think we can allow non-null here unless we want to break compatibility. It doesn't seem necessary to break at this point.